test/libsolidity/semanticTests/array/pop/array_pop.sol

```diff
@@ -15,3 +15,4 @@ contract c {
 // compileViaYul: also
 // ----
 // test() -> 1, 0
+// gas ir: 101390
```

Comment:
Looks like I don't understand what exactly decides what kind of gas measurement is done: why is the legacy gas measurement not defined here too?